### PR TITLE
Added new function to generate: String

### PR DESCRIPTION
**Pull request by huangs1**
Issue #26 : Generating a string value
Issue #10 : Test case for string data type

### CODE REVIEW

**Comment by Lancasterwu** [comment]
I think the issue is probably due to the program got confused between char and string. I'll try some code during Thanksgiving to see if we can fix this.

**Comment by huangs1** [comment]
@Lancasterwu I'm not sure what happened. It originally worked but now there are some errors saying that there is something wrong with the command "pipenv install --dev --three"

File: tada/util/generate.py [hunk 1]
```diff
@@ -19,13 +20,22 @@ def generate_data(chosen_types, chosen_size):
         generated_values = generated_values + (generated_value,)
     return generated_values
 
-
+# pylint: disable=unused-argument
 def generate_int(chosen_size):
     """Generate an int value"""
-    return int(chosen_size)
+    # for i in chosen_size:
```
Comment (lussierc) [inline tada/util/generate.py]: Are these comments needed?

Comment (lussierc) [reply tada/util/generate.py]: Specifically, the comments from lines 26-28.

This looks like old code that you are just not using anymore. We should probably not include this in the pull request.

Comment (huangs1) [reply tada/util/generate.py]: Thanks Christian. I have removed the comments already!

**Comment by lussierc** [comment]
Looks good, I would have approved it but you already merged it.

Thanks for the changes @huangs1 !

**Pull request merged**